modules/openapi/tests/src/DebugSpec.scala

```diff
@@ -63,7 +63,7 @@ final class DebugSpec extends munit.FunSuite {
   }
 
   test("load with debug leaves validation events untouched") {
-    testFilteredErrors(debug = true, expectedCount = 5)
+    testFilteredErrors(debug = true, expectedCount = 4)
```

Comment:
Assuming this is due to smithy model library being updated (transitively in alloy) to no longer output a debug event in some circumstance